kit/typing.py

```diff
@@ -22,3 +24,15 @@ def __call__(self, func: _F) -> _F:
         super_method = getattr(self.interface, func.__name__, None)
         assert super_method is not None, f"'{func.__name__}' does not exist in {self.interface}"
         return func
+
+
+def parsable(func: _F) -> _F:
+    """Mark an object's __init__ as parsable by Configen, so only pre-3.9 type annotations should be used."""
+    assert func.__name__ == "__init__", "@parsable can only be used to decorate __init__."
```

Comment:
does configen not work with functions?

Reply:
seems not

Reply:
I don't know. I think we only use it with objects